index.js

```diff
@@ -0,0 +1,104 @@
+'use strict'
+
+const Hp = require('hemera-plugin')
+const AWS = require('aws-sdk')
+const DynamoStore = require('./store')
+const StorePattern = require('hemera-store/pattern')
+const Joi = require('joi')
+
+function hemeraDynamoStore(hemera, opts, done) {
+  let topic = 'dynamo-store'
+
+  var dynamo = new AWS.DynamoDB(opts.dynamodb)
+  var db = new AWS.DynamoDB.DocumentClient(opts.dynamodb, dynamo)
+
+  function createDb() {
+    return dynamo
+  }
+
+  hemera.decorate('dynamoStore', {
+    createDb
+  })
+
+  hemera.add(StorePattern.create(topic), function (req, cb) {
+    const store = new DynamoStore(db)
+    store.create(req, cb)
+  })
+
+  hemera.add({
+    topic,
+    cmd: 'removeById',
+    id: Joi.required(),
+    collection: Joi.string().required()
+  }, function (req, cb) {
+    const store = new DynamoStore(db)
+    store.removeById(req, cb)
+  })
+
+  hemera.add({
+    topic,
+    cmd: 'updateById',
+    id: Joi.required(),
+    collection: Joi.string().required(),
+    UpdateExpression: Joi.string().required,
```

Comment:
Please wrap it in a seperate property like `options`. The reason why is that every primitive value is relevant for pattern matching. A good practice is to wrap options in an object to exclude it.

```js
hemera.add({
  topic: 'math',
  cmd: 'foo',
  collection: 'bar',
  options: {
   ....  
  }
})
```